koku/api/dataexport/uploader/__init__.py

```diff
@@ -2,7 +2,7 @@
 from abc import ABC, abstractmethod
 
 import boto3
-
+from django.conf import settings
```

Comment:
The linter wants an extra line after this so that there are two blank lines between imports and the first class.